docs/src/cirrus/10_intro.rst

```diff
@@ -10,9 +10,10 @@ both a cost-efficient and performance-efficient manner, regardless if
 processing takes seconds, hours, or longer.
 
 Cirrus consists of two primary pieces:
-- ``cirrus``, a CLI-based project management and deploy tool, and
-- ``cirrus-lib``, a Python library providing a number of useful
-abstractions solving common needs for users writing their own Cirrus components.
+
+- ``cirrus`` -  A CLI-based project management tool.
+- ``cirrus-lib`` - A Python library providing a number of useful
+  abstractions solving common needs for users writing their own Cirrus components.
```

Comment:
@ircwaves @jkeifer Regarding `cirrus-lib` providing...

>a number of useful abstractions solving common needs for users writing their own Cirrus components

Is this correct anymore? I'm not seeing where pulling in `cirrus.lib` when building a Task, Workflow, or Feeder will be useful. As far as Functions, `cirrus.lib` directly supports the built-in functions in `cirrus.lambda_functions`. Do we have use-cases for users creating lambda functions outside of the built-ins?

Reply:
Cirrus.lib could be useful for building feeders, actually. But anything that's in here for building feeders could and probably should be extracted into a utils package (like boto3utils) or is already part of aws-lambda-powertools.